components/FilterComponent.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="text-center">
+    <div v-if="!filterData.active">
```

Comment:
Since the whole component is a big `if` else it should probably be 2 different components